includes/definitions/halon.yaml

```diff
@@ -0,0 +1,9 @@
+os: halon 
+text: Halon 
+type: appliance 
+over:
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_ucd_memory, text: 'Memory Usage' }
+discovery:
+    - sysDescr:
```

Comment:
oops, should have checked this, but you could have added the freebsd sysobjectid as well to reduce the possibility of incorrect matches.